shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/OracleVisitor.java

```diff
@@ -319,21 +319,11 @@ private PredicateSegment createInSegment(final PredicateContext ctx) {
         return result;
     }
     
-    private PredicateBracketValue createBracketValue(final PredicateContext ctx) {
```

Comment:
Hi @xiyelife  
Since there are modifications for #4296, could you give the related changes a look? Thanks